core/tests/test_app.py

```diff
@@ -165,7 +165,7 @@ async def test_handler(sender):
             self.app,
             "loop:call_soon_threadsafe",
             handler=test_handler,
-            args=(self.app,),
```

Comment:
This is a case where the *test* was wrong. #1833 cleaned up handlers, making the first argument empty; but the implementation of `assertActionPerformedWith` wasn't raising an error if a method was called with the wrong arguments. As long as the method was called *at all*, it would pass. 

I picked this up when I was porting the base widget tests - there was a test that should have been failing when I was in an intermediate state, but it wasn't; once the test assertion was fixed, this test also failed.